spark/src/main/scala/org/apache/mahout/sparkbindings/package.scala

```diff
@@ -214,11 +223,11 @@ package object sparkbindings {
           // During maven tests, the maven classpath also creeps in for some reason
           !n.matches(".*/.m2/.*")
         )
-    /* verify jar passed to context
-    log.info("\n\n\n")
-    mcjars.foreach(j => log.info(j))
-    log.info("\n\n\n")
-    */
+    /* verify jar passed to context */
+//    info("\n\n\n")
```

Comment:
this IMO is a useful info (perhaps should be downgraded to debug and be made a little bit more verbose explaining what it is it prints)